release/pkg/assets_capv.go

```diff
@@ -79,7 +87,7 @@ func (r *ReleaseConfig) GetCapvAssets() ([]Artifact, error) {
 	for _, manifest := range manifestList {
 		var sourceS3Prefix string
 		var releaseS3Path string
-		latestPath := r.getLatestUploadDestination()
+		latestPath := getLatestUploadDestination(sourcedFromBranch)
```

Comment:
does CAPV not need the same build component version logic as capd/capa?

Reply:
Great question, we do it in the [vsphere bundle assets](https://github.com/aws/eks-anywhere/blob/main/release/pkg/assets_vsphere.go#L34-L36) file.